specs/altair/light-client/full-node.md

```diff
@@ -103,42 +111,30 @@ def create_light_client_update(state: BeaconState,
     assert hash_tree_root(attested_header) == hash_tree_root(attested_block.message) == block.message.parent_root
     update_attested_period = compute_sync_committee_period_at_slot(attested_block.message.slot)
 
+    update = LightClientUpdate()
+
+    update.attested_header = block_to_light_client_header(attested_block)
+
     # `next_sync_committee` is only useful if the message is signed by the current sync committee
     if update_attested_period == update_signature_period:
-        next_sync_committee = attested_state.next_sync_committee
-        next_sync_committee_branch = compute_merkle_proof_for_state(attested_state, NEXT_SYNC_COMMITTEE_INDEX)
-    else:
-        next_sync_committee = SyncCommittee()
-        next_sync_committee_branch = [Bytes32() for _ in range(floorlog2(NEXT_SYNC_COMMITTEE_INDEX))]
+        update.next_sync_committee = attested_state.next_sync_committee
+        update.next_sync_committee_branch = \
+            compute_merkle_proof_for_state(attested_state, NEXT_SYNC_COMMITTEE_INDEX)
```

Comment:
try to avoid using `\` to break lines
```suggestion
        update.next_sync_committee_branch = compute_merkle_proof_for_state(
            attested_state, NEXT_SYNC_COMMITTEE_INDEX)
```